### PR TITLE
Talise MCS sync

### PR DESCRIPTION
**Pull request by buha**
Various adrv9009 related improvements:
- add some scripts to visualize the DAC_DMA_EXAMPLE data
- get the talise_setup procedure in sync with linux (FRAMER_B was uninitialized in the old procedure!)
- fix DAC_DMA_EXAMPLE (cyclic tx DMA and pl-fifo were overwriting each other)
- MCS sync seems to be working, i see constant near zero offsets across reboots

### CODE REVIEW

File: projects/adrv9009/src/app/app_talise.c [hunk 1]
```diff
@@ -411,6 +417,235 @@ adiHalErr_t talise_setup(taliseDevice_t * const pd, taliseInit_t * const pi)
...
+
+			/***************************************************/
+			/**** Enable SYSREF to Talise JESD204B Deframer ***/
+			/***************************************************/
+			ret = TALISE_enableSysrefToDeframer(pd, TAL_DEFRAMER_A, 1);
+			if (ret != TALACT_NO_ACTION) {
+				printf("%s:%d (ret %d)\n", __func__, __LINE__, ret);
+				ret = FAILURE;
+			}
+
```
Comment (mirceacaprioru) [inline projects/adrv9009/src/app/app_talise.c:634]: nitpick: extra line

File: projects/adrv9009/src/app/app_talise.c [hunk 2]
```diff
@@ -411,6 +417,235 @@ adiHalErr_t talise_setup(taliseDevice_t * const pd, taliseInit_t * const pi)
...
+		}
+		break;
+	case 9:
+		axi_jesd204_tx_lane_clk_enable(tx_jesd);
+
+		/* RESET CDR */
+		uint8_t phy_ctrl;
+		ret = ADIHAL_spiReadByte(pd->devHalInfo, TALISE_ADDR_DES_PHY_GENERAL_CTL_1, &phy_ctrl);
+		if(ret)
```
Comment (mirceacaprioru) [inline projects/adrv9009/src/app/app_talise.c:644]: space after 
```
if (condition)
```

**Comment by dbogdan** [comment]
The only comment is to remove the scripts from here.

**Comment by buha** [comment]
V2:
- removed .tcl and csvplot scripts commits, will be made more generic and moved to no-OS/scripts
- addressed comments

**Pull request merged**